### ISSUE TITLE
VHDir dependency not properly copied into ImplementationGuide JSON

### ISSUE THREAD

**Issue by cmoesel** [FHIR/sushi#140]
If this dependency is put in the package.json:
```
    "hl7.fhir.uv.vhdir":"current"
```
the generated ImplementationGuide is missing it.  It should include:
```
    {
      "uri": "http://hl7.org/fhir/uv/vhdir/ImplementationGuide/hl7.core.uv.vhdir",
      "packageId": "hl7.fhir.uv.vhdir",
      "version": "current"
    }
```

**Comment by cmoesel** [comment]
Re-opening until released

**Comment by cmoesel** [comment]
Fixed in 0.6.1
